content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +47,383 @@ permissions:
+  in the cert-manager controller Pod, if those variables are present.
+- [**Shared config and credentials files**](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html):
+  (`~/.aws/config` and `~/.aws/credentials`).
+  where cert-manager loads credentials from files are mounted into the cert-manager controller Pod.
+
+The advantage of ambient credentials is that they are easier to set up, well
+documented, and AWS provides ways to automate the configuration.
+The disadvantage of ambient credentials is that they are globally available to
+all ClusterIssuer and all Issuer resources, which means that in a multi-tenant
+environment, any tenant who has permission to create Issuer or ClusterIssuer may
+use the ambient credentials and gain the permissions granted to that account.
+
+> 📖 Read [AWS SDKs and Tools standardized credential providers](https://docs.aws.amazon.com/sdkref/latest/guide/standardized-credentials.html)
+> to learn how cert-manager, which uses the AWS SDK for Go V2, supports all these ambient credential sources.
+>
+> ⚠️ By default, cert-manager will only use ambient credentials for
+> `ClusterIssuer` resources, not `Issuer` resources.
+>
+> This is to prevent unprivileged users, who have permission to create Issuer
+> resources, from issuing certificates using credentials that cert-manager
+> incidentally has access to.
+> ClusterIssuer resources are cluster scoped (not namespaced) and only platform
+> administrators should be granted permission to create them.
+>
+> ⚠️ It is possible (but not recommended) to enable ambient authentication mechanisms
+> for `Issuer` resources, by setting the `--issuer-ambient-credentials` flag on
+> the cert-manager controller to true.
 
-cert-manager supports two ways of specifying credentials:
+Here is an example of a `ClusterIssuer` for using Route53 ambient credentials:
 
-- explicit by providing an `accessKeyID` or an `accessKeyIDSecretRef`, and a `secretAccessKeySecretRef`
-- or implicit (using [metadata
-  service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html)
-  or [environment variables or credentials
-  file](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    ...
+    solvers:
+    - dns01:
+        route53 {}:
+```
 
-cert-manager also supports specifying a `role` to enable cross-account access
-and/or limit the access of cert-manager. Integration with
-[`kiam`](https://github.com/uswitch/kiam) and
-[`kube2iam`](https://github.com/jtblin/kube2iam) should work out of the box.
+> ℹ️ Regardless of which ambient mechanism you use, the `route53` section is left empty,
+> because cert-manager can find the credentials, role, and region by looking for environment variables
+> which will be added to the cert-manager Pod.
```

Comment:
In https://github.com/cert-manager/cert-manager/issues/4053#issuecomment-892720234 @viatcheslavmogilevsky pointed out that the role is optional when using IRSA (or any of the ambient credential mechanisms) and I've tried to make that clear here.

@viatcheslavmogilevsky, @austince, @mamachanko please read this new content and tell me if you think it  helps clarify the configuration options:
 * https://deploy-preview-1555--cert-manager.netlify.app/docs/configuration/acme/dns01/route53/#ambient-credentials

Reply:
commented here https://github.com/cert-manager/website/pull/1555/files#r1792533817, lgtm overall